app/screens/global_drafts/components/draft_and_scheduled_post_swipe_actions.tsx

```diff
@@ -109,25 +114,31 @@ const DraftSwipeActions: React.FC<Props> = ({
         DeviceEventEmitter.emit(Events.DRAFT_SWIPEABLE, item.id);
     }, [item.id]);
 
-    const deleteDraft = useCallback(() => {
+    const deletePost = useCallback(() => {
+        if (draftType === DRAFT_TYPE_SCHEDULED) {
+            // TODO: Add swipeable delete for schedule post
```

Comment:
This will be handled in the subsequent PR. https://mattermost.atlassian.net/browse/MM-62200